src/views/players/player/player_info.cpp

```diff
@@ -119,6 +124,33 @@ namespace big
 					net_player_data->m_external_port).data());
 			}
 
+			ImGui::Separator();
+
+			auto id = g_player_service->get_selected()->id();
+
+			if (id != -1)
+			{
+				auto& stats = scr_globals::gpbd_fm_1.as<GPBD_FM*>()->Entries[id].PlayerStats;
+				auto& boss_goon = scr_globals::gpbd_fm_3.as<GPBD_FM_3*>()->Entries[id].BossGoon;
+
+				if (boss_goon.Language >= 0 && boss_goon.Language < 13)
+					ImGui::Text("Language: %s", languages[scr_globals::gpbd_fm_3.as<GPBD_FM_3*>()->Entries[id].BossGoon.Language].name);
+
+				ImGui::Text("CEO Name: %s", boss_goon.GangName);
+				ImGui::Text("MC Name: %s", boss_goon.MCName);
+				ImGui::Text("Money In Wallet: %d", stats.WalletBalance);
+				ImGui::Text("Money In Bank: %d", stats.Money - stats.WalletBalance);
+				ImGui::Text("Total Money: %d", stats.Money);
+				ImGui::Text("Rank: %d (RP %d)", stats.Rank, stats.RP);
+				ImGui::Text("K/D Ratio: %f", stats.KdRatio);
```

Comment:
Would be useful to add `KillsOnPlayers` and `DeathsByPlayers` because K/D doesn't work anymore.